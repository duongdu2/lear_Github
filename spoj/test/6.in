49
27
7215 663 2460 8588 1603 798 3320 7137 6610 7234 4603 5224 1238 7745 3117 2525 5777 6193 4267 6384 6878 1856 9557 5699 2776 7046 3267 
21
1319 9643 7873 9825 6103 5287 3344 2298 9316 5647 721 1758 2711 2117 4261 6078 2900 790 7753 3850 1996 
38
6548 9338 9480 6649 6784 7271 575 7828 3317 811 1822 5969 3057 9879 6583 5225 3395 9270 5076 4503 4960 7606 4728 8668 1694 3590 7186 125 1679 5582 4962 3566 8442 964 8363 7652 1667 1887 
16
7546 3744 2040 4971 7848 7178 5980 5062 7893 8788 2527 7418 6108 231 6498 3584 
29
6003 9342 8798 1323 9145 5079 8827 6763 5490 9901 2268 6604 666 4048 9867 50 3929 4562 6549 2276 9302 5698 1425 6429 1599 9826 6472 713 5106 
47
3334 7089 6648 5897 1414 4550 2812 8120 658 4073 6927 1182 7255 1426 1435 5983 507 4821 2818 821 5721 2361 545 2780 4520 8334 5470 6159 3817 6525 7023 7794 7205 847 4806 3500 2217 2042 6655 8413 4682 8728 814 9335 8930 170 5606 
48
8819 1967 8104 2725 9763 7644 5936 3410 5776 832 5818 2158 6374 1034 1906 3676 9762 9151 2536 6778 3220 7421 7914 3604 3392 1237 5842 8195 1205 1214 3821 8677 112 81 6014 3635 2124 3003 8935 9307 6174 5574 2577 6716 9583 1104 2087 8486 
19
9275 9223 3145 6428 3914 847 677 3374 2124 3004 1591 2738 1199 210 2133 7202 1310 6929 742 
50
8154 6390 6815 6124 1046 9087 1808 3931 5787 9118 6462 3074 3852 6369 9665 7354 969 3270 9728 4465 5857 4292 7793 307 9942 1350 4262 4107 6431 9721 8225 1237 5213 3809 7914 9545 6382 8743 468 2435 8581 3143 9110 9574 6559 547 5397 5575 997 5108 
14
3409 4863 6011 3419 3944 8275 8975 3563 3804 12 9515 1579 8564 8174 
32
6238 8892 9850 1021 6947 1619 3005 230 1134 7780 593 7719 1176 6490 7382 1047 213 775 8931 4585 6155 4250 2070 4898 7176 9980 9737 5530 8963 5707 5358 2354 
33
5626 7360 626 6481 5772 3610 6910 2406 1683 7777 1747 914 5579 7858 4575 9039 8314 1887 8972 9159 2951 6307 9401 8841 805 2008 3537 6491 9419 2668 7916 922 1722 
46
5798 514 9104 1500 3485 1880 8197 8143 3871 9168 1856 1004 5944 346 1461 688 6975 4436 30 81 2125 892 3279 3337 725 9474 8524 8863 517 674 5405 5919 5806 4654 1080 5242 3580 6417 9772 8523 43 3432 5792 1850 3636 3647 
38
2497 7976 4858 8885 543 108 6716 9361 1460 2414 7848 4292 707 5414 2451 8918 7738 2663 8508 3042 1797 124 3115 7244 3707 3275 9975 7290 6283 2053 5943 205 9591 6733 2839 2792 165 482 
33
6954 8007 4733 3973 228 129 698 3155 6114 8688 5651 5578 3757 9599 672 189 9078 8031 2333 4077 1991 1591 3018 6227 837 2445 363 833 8555 584 9590 6503 7102 
8
9340 6607 9819 6494 4460 6909 6667 4824 
17
2997 9360 7263 5434 4209 7706 4307 5950 2497 7081 409 1530 1711 6891 3432 1395 7259 
50
8550 5810 4672 9485 3289 6866 3431 7472 39 3305 3599 9817 7747 2164 6915 7850 1888 1569 5315 853 4506 4652 1036 5876 3128 9962 2256 6240 766 5058 936 5043 9894 7882 2664 2512 2153 9857 5662 8141 5094 5264 7808 5398 2023 5328 2751 6227 6354 4591 
10
2511 2416 6730 1100 8853 3383 5577 8639 3197 2971 
13
5737 7157 9162 6954 8724 4184 7827 3634 1687 8519 8697 6139 3174 
16
5962 1570 7688 9697 4220 9399 3619 5560 5621 3626 3675 8560 8499 915 7681 2838 
38
8683 6366 3909 6288 9080 4600 4002 2471 1700 2241 1106 6998 2213 4593 7102 8948 6819 385 1640 4326 7092 6117 7269 1484 6141 3996 3910 5582 9818 8228 6890 6941 1120 6014 1702 3414 2688 9921 
12
3400 1636 8760 6607 9844 6850 4314 9847 6723 3872 278 2727 
15
2320 1244 8160 8882 5431 2769 4633 5478 7329 5092 9538 7460 9672 648 2314 
49
4646 9672 9125 9874 935 2329 5561 724 2783 932 6266 9224 153 6172 8852 1757 6013 5773 3709 4682 6393 3983 7468 7603 9483 5624 9059 8618 3855 3347 6212 2424 484 9189 4816 7629 6408 6050 3084 5937 5702 9370 9794 7360 2808 2604 4266 2720 2296 
11
4605 8402 7958 9080 7346 4414 8483 201 2481 9866 2032 
42
3302 4791 4702 645 9436 5641 48 1742 6492 7752 8954 3362 8845 1506 3075 8729 1999 4252 7469 9356 3095 1813 340 4840 2373 3445 5556 8962 2762 6426 1626 9434 5599 1684 1870 4697 5531 8738 9002 4360 826 7513 
30
8599 308 6054 873 5221 5529 123 396 4792 4261 3910 4529 6329 3493 7017 3797 864 5297 3725 5190 1079 8629 4980 8998 2781 7367 2289 9861 158 6910 
24
8427 6951 6280 7644 9888 8400 1744 7350 8271 4599 3345 8342 6611 2444 9876 7286 4404 7295 9370 8007 9395 5348 8501 2034 
42
8643 3020 5357 8447 4490 6715 8777 1217 1505 9330 7601 6338 6539 9019 9550 9945 4581 8818 1543 1621 8160 5683 7888 966 9900 1700 9602 9609 5238 2088 218 1235 198 9269 3909 8644 7462 6665 2561 5706 5534 2241 
22
7066 595 4825 4583 7237 9948 2960 9462 4850 3203 2233 9984 799 6286 7217 7719 4622 8750 8590 6374 9125 1836 
32
4614 1572 6442 1735 7677 5422 1313 6370 4764 9266 9701 4772 7045 7332 6712 4528 1270 6471 9892 5433 6046 1453 942 650 337 1114 9462 4149 7585 6348 601 2236 
16
7405 1310 7934 6773 5312 9786 2496 8451 1363 9719 6284 7168 7589 9437 2316 8053 
44
5378 8687 6133 5186 7490 5858 473 5237 9914 5809 8889 9039 6336 398 9947 4153 7562 9889 168 1720 1156 7550 280 4096 1276 9463 7909 4050 2506 9374 1147 5437 8716 5386 7109 9416 7979 851 3012 6167 5423 358 2547 6382 
41
6519 946 2825 2597 5900 5422 4688 2134 1374 1096 9287 9785 1262 4688 3572 1468 4729 5574 8370 6344 538 3549 8786 1923 3636 1183 4787 7033 499 9779 5885 4669 4520 4458 3582 5179 8287 4523 1318 4072 4099 
11
1107 2147 7674 1967 8471 3929 1017 4839 862 5178 9040 
3
2920 1307 2553 
33
7729 7012 4865 7031 2282 8313 6926 5027 2046 8546 3710 5534 2955 3030 1116 3424 3938 6134 8579 787 5763 5800 9236 1480 8995 8522 5514 198 7386 4574 5467 2183 8028 
40
1947 5889 8993 6983 4116 2452 1664 108 5741 4857 6156 7894 5467 2010 1504 8719 199 7790 808 2847 4324 6687 2285 8670 7635 1379 5289 4118 1958 8327 9560 5757 2916 257 5198 3653 1244 3504 4438 4673 
20
9716 106 5354 4911 2286 3117 8876 3731 7063 8353 9663 4584 825 7527 601 7095 7264 33 5576 27 
45
3389 282 9087 7380 9103 1143 3062 2563 5375 8361 9630 9743 5091 2913 379 2941 6169 757 2727 3621 6970 9602 1317 2272 2335 5208 524 1640 6802 3375 7988 8732 984 8192 282 1533 1584 3415 8929 7047 9049 6285 2436 2095 1760 
12
9242 4914 3626 5409 829 2149 3467 2436 8537 5543 2967 5222 
49
2545 7429 3894 28 8749 68 3379 5782 8168 5821 8010 3993 3110 8469 4509 7776 3411 8314 5969 2709 1074 2053 7865 5442 2252 6607 3620 4973 9137 6727 1924 9284 2531 496 3245 1445 8048 2204 5356 1990 2194 5715 8577 4181 4635 5011 5574 8693 2211 
30
9726 5573 9480 394 3124 5141 7001 4610 8191 3536 7958 7467 5920 5732 347 7166 3113 8734 8997 5995 262 8647 1868 3614 6492 7406 6405 6648 3146 7430 
38
2514 3769 3729 7350 717 641 8194 1374 7380 505 8184 2482 739 1581 1219 4059 3358 3015 4085 7164 4744 125 7318 3029 8880 3435 3373 9429 5448 5993 8241 2611 3211 8669 8296 2905 2348 6613 
27
4625 6179 4989 2165 474 9049 5355 1041 4003 4935 3942 7643 4890 4205 5082 2041 7022 5180 4378 9643 2121 1775 3122 2130 9090 7148 8457 
37
1978 3438 6191 2134 5237 2792 6386 8171 3982 3428 7646 5390 5943 803 6669 3242 9036 6684 7671 5335 8046 3259 9700 6005 7035 8853 4822 8206 4634 4401 3109 4117 4137 1563 8634 6486 5782 
45
2665 9420 4607 626 9668 9555 2390 6296 4152 17 9667 5814 3140 5144 8191 482 5323 2669 4737 9763 3978 1702 1101 940 8474 3645 6542 4253 1102 6744 960 5248 3034 5993 4649 4641 8795 2214 4642 2506 8360 496 7119 6627 6400 
46
7035 6889 2647 8980 9324 6906 3512 9648 1451 6379 3664 9893 3771 4102 564 3470 3585 3450 9955 6911 7209 2698 7808 4326 8158 2841 7133 6144 4989 8270 9330 969 6714 3108 2853 8953 6687 3287 2459 7142 896 1659 7013 8085 580 7245 
