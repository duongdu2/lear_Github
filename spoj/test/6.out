25890
28784
28697
24529
29594
26993
29108
25700
29391
27054
29567
27979
28414
28927
28267
26068
24329
29713
24222
26583
27656
28819
28451
28092
29340
27429
28226
27488
29159
29454
29394
29055
28942
29750
28851
25185
6780
26810
27272
28255
28975
23322
27170
28203
26978
27782
27589
29098
29496
